plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduIntegrationGroupedExecution.java

```diff
@@ -98,6 +98,43 @@ public void testGroupedExecutionJoin()
         assertUpdate("DROP TABLE test_grouped_execution_t2");
     }
 
+    @Test
+    public void testGroupedExecutionJoinRangePartition()
+    {
+        assertUpdate("CREATE TABLE IF NOT EXISTS test_grouped_execution_range_t1 (" +
```

Comment:
It is always better to use table with random prefix, so that things won't fail. Please check the usage of io.trino.testing.sql.TestTable.randomTableSuffix